model/src/main/java/com/ge/predix/acs/model/Obligation.java

```diff
@@ -0,0 +1,113 @@
+/*******************************************************************************
+ * Copyright 2017 General Electric Company
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************/
+
+package com.ge.predix.acs.model;
+
+import java.util.Collections;
+import java.util.List;
+
+//import javax.validation.constraints.Max;
+//import javax.validation.constraints.Min;
+//import javax.validation.constraints.NotNull;
+
+import io.swagger.annotations.ApiModel;
+
+/**
+ * @author Sebastian Torres Brown
+ * 
+ *         Keti obligations model defined by XACML 3.O
+ *
+ */
+@ApiModel(
+        description = "Obligation augments the interface contract between the policy engine (PDP) and "
+                + "enforcement point (PEP).  In addition to the actual access decision of either 'Permit' or 'Deny', "
+                + "the policy engine may return some supplementary information to the enforcement point in the form"
+                + " of an Obligation. Obligations are a set of operations that must be performed by the PEP "
+                + "in conjunction with an authorization decision, e.g., 'Permit, and in addition fulfill the "
+                + "indicated Obligation(s)' or, alternatively: 'Deny, and in addition fulfill the indicated"
+                + " Obligation(s)'.  If the PEP does not understand, or cannot fulfill, any of the obligations, "
+                + "then it MUST act as if the PDP had returned a 'Deny' authorization decision value.")
+public class Obligation {
+
+    private String id;
+
+    private ObligationType type;
+
+    private Object actionTemplate;
```

Comment:
jsonnode fails on the openjpa layer when trying to save,  the rest controller layer makes sure the serialization into object is valid, if you send an invalid json object it will throw an erro 400.